{"Functions": {"AssertionError()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with no detail message.", "history": "Added in API level 1", "FullName": "public AssertionError ()"}, "AssertionError(boolean)": {"Returns": [], "Parameters": [["boolean", "boolean: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified boolean, which is converted to a string as defined in section 15.18.1.1 of", "history": "Added in API level 1", "FullName": "public AssertionError (boolean detailMessage)"}, "AssertionError(char)": {"Returns": [], "Parameters": [["char", "char: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified char, which is converted to a string as defined in section 15.18.1.1 of", "history": "Added in API level 1", "FullName": "public AssertionError (char detailMessage)"}, "AssertionError(double)": {"Returns": [], "Parameters": [["double", "double: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified double, which is converted to a string as defined in section 15.18.1.1 of", "history": "Added in API level 1", "FullName": "public AssertionError (double detailMessage)"}, "AssertionError(long)": {"Returns": [], "Parameters": [["long", "long: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified long, which is converted to a string as defined in section 15.18.1.1 of", "history": "Added in API level 1", "FullName": "public AssertionError (long detailMessage)"}, "AssertionError(float)": {"Returns": [], "Parameters": [["float", "float: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified float, which is converted to a string as defined in section 15.18.1.1 of", "history": "Added in API level 1", "FullName": "public AssertionError (float detailMessage)"}, "AssertionError(int)": {"Returns": [], "Parameters": [["int", "int: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified int, which is converted to a string as defined in section 15.18.1.1 of", "history": "Added in API level 1", "FullName": "public AssertionError (int detailMessage)"}, "AssertionError(java.lang.Object)": {"Returns": [], "Parameters": [["@B_java#lang#Object_E@", "Object: value to be used in constructing detail message"]], "Throws": [], "SeeAlso": ["@B_java#lang#Throwable#getCause_E@"], "Permissions": [], "Description": "Constructs an AssertionError with its detail message derived from the specified object, which is converted to a string as defined in section 15.18.1.1 of If the specified object is an instance of Throwable, it becomes the", "history": "Added in API level 1", "FullName": "public AssertionError (Object detailMessage)"}, "AssertionError(java.lang.String,java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: the detail message, may be null"], ["@B_java#lang#Throwable_E@", "Throwable: the cause, may be null"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs a new AssertionError with the specified detail message and cause. Note that the detail message associated with cause is", "history": "Added in API level 19", "FullName": "public AssertionError (String message, Throwable cause)"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Error_E@"], "ClassName": "java.lang.AssertionError", "ClassDesc": "Thrown to indicate that an assertion has failed. The seven one-argument public constructors provided by this class ensure that the assertion error returned by the invocation:"}